de-lib/src/main/java/org/iplantc/de/theme/base/client/notifications/JoinTeamRequestDefaultAppearance.java

```diff
@@ -16,75 +16,6 @@ public JoinTeamRequestDefaultAppearance(JoinTeamRequestDisplayStrings displayStr
         this.displayStrings = displayStrings;
     }
 
-    @Override
-    public String joinTeamRequestHeader() {
-        return displayStrings.joinTeamRequestHeader();
```

Comment:
I may have missed it, but all the deletions in this file should hopefully mean deletions in whatever file `displayStrings` is too